src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.CachedDocumentSearch.cs

```diff
@@ -101,7 +101,7 @@ private static void ClearCachedData()
         ImmutableArray<DocumentKey> priorityDocumentKeys,
         string searchPattern,
         IImmutableSet<string> kinds,
-        Func<ImmutableArray<RoslynNavigateToItem>, Task> onItemsFound,
+        Func<ImmutableArray<RoslynNavigateToItem>, VoidResult, CancellationToken, Task> onItemsFound,
```

Comment:
need the VoidResult for the 'args'.  we don't actually end up needing any args ourselves, so this is how we just indicate it's unused.

also, i found that things wer ecleaner if i made it so that 'cancellation' was consistently passed along as an actual arg to the lambdas, instead of needing to package it in the 'args' parameter.